example/android/android-studio-robolectric-example/.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: android
```

Comment:
does it make sense to still keep this file, when the rest of the project is using circleci?